src/server/raft_client.rs

```diff
@@ -862,7 +864,16 @@ async fn start<S, R>(
                 .report_store_unreachable(back_end.store_id);
             continue;
         } else {
-            debug!("connection established"; "store_id" => back_end.store_id, "addr" => %addr);
+            let wait_conn_duration = begin.elapsed();
```

Comment:
If the connection is estabilished after serveral round "wait connect timeout", this kind of wait seems not to be recorded, should we add them or add some "wait connect timeout" counter too?

Reply:
nice catch, done.